app/assets/javascripts/reports/new.js

```diff
@@ -1034,6 +1048,25 @@ function reportHandsonTableConverter() {
         // TODO
       });
     });
+
+    $('.reports-new').on('click', '#saveAsNewReport', function(e) {
+      var params = getReportData();
+      params.report.name = 'New ' + params.report.name;
+      e.preventDefault();
+      $.post(this.dataset.createUrl, params, function() {
+        // TODO
```

Comment:
Shouldn't it be implemented?

Reply:
We have a ticket for front-end validations (check report name and selected content). I think it can be done in the scope of this ticket.

Reply:
Ok